File: backendworker/NHSOnline.Backend.Worker/GpSystems/Suppliers/Tpp/Models/Appointments/Slot.cs
using System;
using System.Xml.Serialization;

namespace NHSOnline.Backend.Worker.GpSystems.Suppliers.Tpp.Models.Appointments
{
    [Serializable]
    public class Slot
    {
        [XmlAttribute("startDate")]
        public string StartDate {get; set;}

        [XmlAttribute("endDate")]
        public string EndDate { get; set; }

        [XmlAttribute("type")]
        public string Type { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/tpp/models/Slot.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name = "Slot")
@XmlAccessorType(XmlAccessType.FIELD)
data class Slot {
    @XmlAttribute var startDate: String = "default startDate",
    @XmlAttribute var endDate: String = "default endDate",
    @XmlAttribute var type: String = "default type"
}